pkg/oc/cli/admin/policy/review.go

```diff
@@ -195,80 +230,55 @@ func CheckStatefulSetWithWolumeClaimTemplates(obj runtime.Object) error {
 	// spec.template.spec validateContainers to validate volumeMounts
 	//https://github.com/openshift/origin/blob/master/vendor/k8s.io/kubernetes/pkg/apis/apps/validation/validation.go#L57
 	switch r := obj.(type) {
-	case *apps.StatefulSet:
+	case *appsv1beta1.StatefulSet:
```

Comment:
You're gonna need to add all of StatefulSet supported versions: `*kappsv1beta1.StatefulSet, *kappsv1beta2.StatefulSet, *kappsv1.StatefulSet` is what I got from https://github.com/openshift/origin/blob/master/pkg/oc/cli/set/triggers.go

Reply:
Thanks, done